Formula/chrome-export.rb

```diff
@@ -0,0 +1,28 @@
+class ChromeExport < Formula
+  desc "Convert Chrome’s bookmarks and history to HTML bookmarks files"
+  homepage "https://github.com/bdesham/chrome-export"
+  url "https://github.com/bdesham/chrome-export/archive/v2.0.tar.gz"
+  sha256 "ff972c3d88b030b65823388e08cd65ebd8795f0674b5581e9e8fdf3a0f4dc7ef"
+
+  depends_on "python"
+
+  def install
+    bin.install "export-chrome-bookmarks"
+    bin.install "export-chrome-history"
+
+    # Install the test files in case someone wants to run "brew test" later;
+    # compress them first so that they take up as little space as possible.
+    system "tar", "-cJf", "test_files.xz", "test"
```

Comment:
As far as I can see, the tests are ~100 KB, so let's not compress them. It will make the formula simpler.